pages/Documents/ConversationalAI/ConversationOrchestrator/NextActionsAPI/overview.md

```diff
@@ -39,7 +39,7 @@ A typical use case for using the Next Actions API is to get a routing decision t
 v2 of the Conversation Orchestrator APIs was released in July 2021. The new version applies to both the [Conversation Context Service](conversation-orchestrator-conversation-context-service-overview.html) and the Next Actions API. v2 has the following additional benefits:
 
 * Higher scalability
-* Improved Better data security through OAuth2.0 authorization mechanism
+* Improved Better data security through OAuth 2.0 authorization mechanism
```

Comment:
I would consider removing the word "Better" at some point as "Improved" already covers the meaning and I am also unsure why it is capitalized.

Reply:
Good catch—updated!